index.ts

```diff
@@ -93,6 +100,11 @@ async function main() {
                     const additionalLabelsToAdd = projectIssue?.assignee?.login
                     ? ["Unavailable"]
                     : [];
+                    const issuePriceLabel  = getIssuePriceLabel(projectIssue) 
```

Comment:
Pls move the following repeated code to a separate function:
```
const issuePriceLabel  = getIssuePriceLabel(projectIssue);
if(issuePriceLabel === DEFAULT_PRICE_LABEL){
  console.log(`Pricing not set for the issue, skipping. ${projectIssue.html_url}`)
  return
}
```

Reply:
@rndquu Updated.